src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -219,25 +210,18 @@ export const editVenueCastSchema = Yup.object()
 
   // possible locations for the banner image
   .from("config.landingPageConfig.coverImageUrl", "bannerImageUrl")
-  .from("config.landingPageConfig.bannerImageUrl", "bannerImageUrl")
-
-  // possible locations for the map icon
-  .from("config.mapIconImageUrl", "mapIconImageUrl")
-  .from("mapIconImageUrl", "mapIconImageUrl");
+  .from("config.landingPageConfig.bannerImageUrl", "bannerImageUrl");
 
 export const editPlacementCastSchema = Yup.object()
   .shape<Partial<PlacementInput>>({})
 
   // possible locations for the map icon
-  .from("config.mapIconImageUrl", "mapIconImageUrl")
-  .from("mapIconImageUrl", "mapIconImageUrl")
   .from("placement.addressText", "addressText")
   .from("placement.notes", "notes")
   .required();
 
 export const editPlacementSchema = Yup.object().shape<PlacementInput>({
   mapIconImageFile: createFileSchema("mapIconImageFile", false).notRequired(),
```

Comment:
Given we've removed `mapIconImageUrl`, would we not also be wanting to remove `mapIconImageFile`? Since presumably they are going to relate to the same sort of thing?